code/modules/mob/living/simple_animal/hostile/space_dragon.dm

```diff
@@ -434,46 +456,31 @@
  */
 /mob/living/simple_animal/hostile/space_dragon/proc/victory()
 	objective_complete = TRUE
+	rift_empower(TRUE, TRUE)
 	var/datum/antagonist/space_dragon/S = mind.has_antag_datum(/datum/antagonist/space_dragon)
 	if(S)
 		var/datum/objective/summon_carp/main_objective = locate() in S.objectives
 		if(main_objective)
 			main_objective.completed = TRUE
-	sound_to_playing_players('sound/machines/alarm.ogg')
-	sleep(100)
-	priority_announce("A large amount of lifeforms have been detected approaching [station_name()] at extreme speeds. Evacuation of the remaining crew will begin immediately.", "Central Command Spatial Corps")
+	priority_announce("A large amount of lifeforms have been detected approaching [station_name()] at extreme speeds. Remaining crew are advised to evacuate as soon as possible.", "Central Command Wildlife Observations")
```

Comment:
It niggles me a little, but about every other priority announcement about space pests around or within the station uses the "Lifesign Alert" title. I'm not asking you to change it, but I believe announcement titles will need some recoherence in the future in the form of defines.